cmplibrary/src/test/java/com/sourcepoint/cmplibrary/data/network/util/HttpUrlManagerTest.kt

```diff
@@ -233,15 +235,19 @@ class HttpUrlManagerTest {
 
     @Test
     fun `GIVEN a PROD env getChoiceUrl RETURN the prod link`() {
-        val param = ChoiceParamReq(
+        val param = GetChoiceParamReq(
+            choiceType = ChoiceTypeParam.CONSENT_ALL,
             accountId = 22,
+            propertyId = 17801,
```

Comment:
@bohdan-go-wombat can we change the property id to use our "demo" property? The demo property id is `16893`.